ui/app/(prowler)/lighthouse/chat.tsx

```diff
@@ -0,0 +1,136 @@
+"use client";
+
+import { useChat } from "@ai-sdk/react";
+
+import { MemoizedMarkdown } from "@/components/memoized-markdown";
+
+// Add this interface above the Chat component
+interface SuggestedAction {
+  title: string;
+  label: string;
+  action: string;
+}
+
+export default function Chat() {
+  const { messages, input, handleSubmit, handleInputChange, append, status } =
+    useChat({
+      api: "/api/lighthouse/analyst",
+      credentials: "same-origin",
+      experimental_throttle: 100,
+      sendExtraMessageFields: true,
+      onFinish: () => {
+        // Handle chat completion
+      },
+      onError: () => {
+        console.log("An error occurred, please try again!");
+      },
+    });
+
+  const suggestedActions: SuggestedAction[] = [
+    {
+      title: "Are there any exposed S3",
+      label: "buckets in my AWS accounts?",
+      action: "List exposed S3 buckets in my AWS accounts",
+    },
+    {
+      title: "What is the risk of having",
+      label: "RDS databases unencrypted?",
+      action: "What is the risk of having RDS databases unencrypted?",
+    },
+    {
+      title: "What is the CIS 1.10 compliance status",
+      label: "of my Kubernetes cluster?",
+      action:
+        "What is the CIS 1.10 compliance status of my Kubernetes cluster?",
+    },
+    {
+      title: "List my highest privileged",
+      label: "AWS IAM users with full admin access?",
+      action: "List my highest privileged AWS IAM users with full admin access",
+    },
+  ];
+
+  return (
+    <div className="flex h-[calc(100vh-theme(spacing.16))] min-w-0 flex-col bg-background">
```

Comment:
Can you use the `<ContentLayout>` component? I don't think you need a new layout for this. Take a look at any other page.

`<ContentLayout title="Configure Chatbot" icon="lucide:settings">`

If so, please, remove the `app/(prowler)/lighthouse/config/layout.tsx`